src/outfit.ts

```diff
@@ -446,6 +512,18 @@ export class Outfit {
       usedSlots.add(unusedSlot);
     }
 
+    // Handle the rider slots next
+    const bjorn = this.riders.get($slot`buddy-bjorn`);
+    if (bjorn) {
+      if (myEnthronedFamiliar() === bjorn) enthroneFamiliar($familiar.none);
+      if (myBjornedFamiliar() !== bjorn) bjornifyFamiliar(bjorn);
+    }
+    const crown = this.riders.get($slot`crown-of-thrones`);
+    if (crown) {
+      if (myBjornedFamiliar() === crown) bjornifyFamiliar($familiar.none);
+      if (myEnthronedFamiliar() !== crown) enthroneFamiliar(crown);
+    }
+
```

Comment:
I think you need to add the set Bjorn/throne slots to `usedSlots` as well, so they are excluded from the maximizer call below.